File: estruturas condicionais/operadora.c
#include <stdio.h>
int main() {
    int minutos;
    double valor = 50.0;
    printf("Digite a quantidade de minutos:\n");
    scanf("%d", &minutos);
    if (minutos > 100) {
        valor += (minutos - 100) * 2.0;
    }
    printf("Valor a pagar: R$ %.2lf\n", valor);
    return 0;
}

File: c#  lista/Operadora.cs
using System;
class Operadora {
    static void Main() {
        Console.Write("Minutos usados: ");
        int minutos = int.Parse(Console.ReadLine());

        double conta = 50.0;
        if (minutos > 100)
            conta += (minutos - 100) * 2.0;

        Console.WriteLine($"Valor a pagar: R$ {conta:F2}");
    }
}